shuriken/include/shuriken/common/extract_file.h

```diff
@@ -0,0 +1,21 @@
+//--------------------------------------------------------------------*- C++ -*-
+// Shuriken-Analyzer: library for bytecode analysis.
+// @author Farenain <kunai.static.analysis@gmail.com>
+//
+// @file extract_file.h
+// @brief Utility to extract zip files such as APKs and IPAs
+// types
+
+#ifndef SHURIKENLIB_EXTRACT_FILE_H
+#define SHURIKENLIB_EXTRACT_FILE_H
+
+#include "zip.h"
```

Comment:
Try to avoid using external headers like `zip.h` in external files that will be externally exposed, because the user will need to add the path to the zip headers too in their programs for the file expansion.